Mathlib/Analysis/SpecificLimits/Basic.lean

```diff
@@ -110,6 +110,19 @@ theorem tendsto_pow_atTop_nhds_0_of_lt_1 {𝕜 : Type _} [LinearOrderedField 
       (tendsto_inv_atTop_zero.comp this).congr fun n => by simp)
 #align tendsto_pow_at_top_nhds_0_of_lt_1 tendsto_pow_atTop_nhds_0_of_lt_1
 
+theorem tendsto_pow_atTop_nhds_0_iff_lt_1 {𝕜 : Type _} [LinearOrderedField 𝕜] [Archimedean 𝕜]
+    [TopologicalSpace 𝕜] [OrderTopology 𝕜] {r : 𝕜} (h₁ : 0 ≤ r) :
+    r < 1 ↔ Tendsto (fun n : ℕ => r ^ n) atTop (𝓝 0) := by
```

Comment:
Please reverse the order of the `iff`.

bors d+

Reply:
Oops, sorry Floris and I were reviewing simultaneously.

If he has no objections then I'd still like to propose my modification.

Reply:
I am happy to
1. Reverse the `iff`
1. Add the `simp` version with the asbolute value suggested by Oliver, but do you want to replace it or to add it? I think both could be useful.
1. Aside: Is there a webpage where the new convention about PR names is updated>?

Reply:
Thanks @faenuccio

Regarding 2, I contend that the statement I proposed should replace yours.

Regarding 3, I don't know!

Reply:
I like Oliver's suggestion more, too. I think it can replace this one.

3: https://leanprover.github.io/lean4/doc/dev/commit_convention.html

Reply:
Note: with Oliver's suggestion, maybe rename to `tendsto_pow_atTop_nhds_0_iff`